samples/apkman/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.11)
+
+if (NOT UNIX)
```

Comment:
or not SGX?

Reply:
Added the check even though none of the other SGX specific samples do the OE_SGX check.

It looks like we don't publish an openenclave package for OPTEE yet:
https://github.com/openenclave/openenclave/releases/tag/v0.17.1

I'm not sure whether it is an oversight that none of the SGX only samples have an OE_SGX check or whether these samples don't end up in the OPTEE package/workflow.